### ISSUE TITLE
Cluster raft initialisation not working

### ISSUE THREAD

**Issue by didlawowo** [AxaFrance/SlimFaas#46]
### Issue and Steps to Reproduce

raft cluster has no leader

Rebuild the image to get arm64
change cluster role to match correct namespace

### Versions
latest

### Screenshots

<img width="1124" alt="image" src="https://github.com/AxaFrance/SlimFaas/assets/12622760/db9b3170-d195-43d3-80f1-a2706e1227f2">

#### Expected
raft initialisation complete

#### Actual

all 3 statefulset are crying 
Raft cluster has no leader

**Comment by guillaume-chervet** [comment]
Thank you @didlawowo .
What kind of machine are you using?
Does it happen every time when you are starting the namespace?

**Comment by didlawowo** [comment]
it run on k3s with pi5 and arm64 build on my own.

**Comment by didlawowo** [comment]
not sure it's work but ihm show nothing

![image](https://github.com/AxaFrance/SlimFaas/assets/12622760/98a5ba3c-c8a5-4020-8d19-c2e1712912d9)

**Comment by guillaume-chervet** [comment]
May you copy paste your yaml script? @didlawowo

**Comment by guillaume-chervet** [comment]
Hi @didlawowo if you are running the demo or configuration like the demo SlimFaas should be exposed at http://localhost:30021/status-function/fibonacci3 for example.

**Comment by didlawowo** [comment]
> May you copy paste your yaml script? @didlawowo For exemple demo port are not 5000 and defaut route do not answer. Ypu can try /health for exemple.

there is no web interface for managing function ?

**Comment by guillaume-chervet** [comment]
Not yet @didlawowo, but it is a good idea, I may reuse that one : https://github.com/AxaFrance/dailyclean

**Comment by didlawowo** [comment]
yeah i think it's mandatory to have observability with serverless
